tutorials/python/python-video-generation/code/Readme.md

```diff
@@ -0,0 +1,40 @@
+# Local environment set up
+---
+
+## Install Python 3
+
+### Linux and OSX
+Using version manager [asdf-vm](https://asdf-vm.com/#/core-manage-asdf-vm?id=install-asdf-vm):
+
+Install [Python Plugin](https://github.com/danhper/asdf-python):
+`$ asdf plugin-add python https://github.com/danhper/asdf-python.git`
```

Comment:
In this file, I suggest using the markdown code block (triple backtick) to present instructions or commands.

Reply:
Good suggestion, I'll take it :)